mongodbatlas/doc.go

```diff
@@ -33,7 +33,7 @@ pass cancellation signals and deadlines to various services of the client for
 handling a request. In case there is no context available, then context.Background()
 can be used as a starting point.
 
-Authentication
+Authentication.
```

Comment:
[q] is this a linter requirement?

Reply:
yes, the lint test was failing because of this

Reply:
I think this is a bit odd. For example, we don't need the dot in line 18. Could you share the rule that requires this? Wondering if we can remove it/search if it's a bug/try to tweak the text to not require this...

Reply:
you are right. I will try to remove it

Reply:
<img width="841" alt="Screenshot 2022-08-16 at 10 10 57" src="https://user-images.githubusercontent.com/5663078/184830653-82d34c3e-be45-44ce-91e5-0cb7638ef2d9.png">